src/payment/strategies/humm/humm-payment-strategy.ts

```diff
@@ -0,0 +1,90 @@
+import { FormPoster } from '@bigcommerce/form-poster';
+
+import { PaymentStrategy } from '..';
+import { PaymentActionCreator } from '../..';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import { PaymentRequestOptions } from '../../payment-request-options';
+
+export default class HummPaymentStrategy implements PaymentStrategy {
+    constructor(
+        protected _store: CheckoutStore,
```

Comment:
These can be `private` now we're not expected this class to be extended 

(sorry, mistake in my example code!)